mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMessageIdManagerSideEffectTest.java

```diff
@@ -36,7 +37,8 @@
     static CassandraClusterExtension cassandraCluster = new CassandraClusterExtension(MailboxAggregateModule.MODULE);
 
     @Override
-    protected MessageIdManagerTestSystem createTestSystem(QuotaManager quotaManager, EventBus eventBus, Set<PreDeletionHook> preDeletionHooks) {
+    protected MessageIdManagerTestSystem createTestSystem(QuotaManager quotaManager, InVMEventBus eventBus, Set<PreDeletionHook> preDeletionHooks) {
+        eventBus.initialize();
```

Comment:
don't we initialize others EventBuses?

Reply:
Actually, the fact that we can't rely on typing to know if things are initialized or not makes this PR unmanageable.

I plan to drop it all together in favour of https://github.com/linagora/james-project/pull/3292 (waiting a :green_apple: build there)